src/default-config.ls

```diff
@@ -0,0 +1,53 @@
+require! <[ path fs ]>
+
+{ filter, map, first, join, keys } = require 'prelude-ls'
+
+# RC automatically overwrites these with env variables.
+# For example to edit environment set arch_environment
+
+/* Map of parsers which take a file path and parse functions*/
+
+parsers =
+  js: -> require it,
+  ls: -> require it
+
+parser = (fname) -> parsers[(path.extname fname).slice(1)](fname)
+
+fpath-regex = new RegExp "arch\.config\.(?:#{parsers |> keys |> join '|'})$"
+
+filter-configs = -> fpath-regex.test it
+
+merge = (x, xs) -> x import xs
+
+initial-conf =
+  app-path:     process.env.arch_app_path or path.resolve '.'
```

Comment:
Who are all these default `env` overrides for? You can just as well just drop your own `arch.config.js`which does the `env` reading if you need it. Why don't we keep it as simple as possible?